src/pages/en/migrate.md

```diff
@@ -42,6 +42,12 @@ Astro no longer supports components or JSX expressions in Markdown pages by defa
 
 To make migrating easier, a new [legacy flag](/en/reference/configuration-reference/#legacyastroflavoredmarkdown) can be used to re-enable previous Markdown features.
 
+### `<Markdown />` Component Removed
+
+If you were previously using Astro's built-in `<Markdown />` component, this has been moved to a separate package. You will now need to install `@astrojs/markdown` and update your imports accordingly. For more details, see [the `@astrojs/markdown` README](https://github.com/withastro/astro/tree/main/packages/markdown/component).
+
+Astro now has support for [MDX](https://mdxjs.com/) through our [MDX integration](https://github.com/withastro/astro/tree/main/packages/integrations/mdx). MDX gives you the ability to create components that can be imported into an Astro page that include both Markdown and component usage. MDX make a good alternative for the Markdown component due to it's large community and stable APIs.
```

Comment:
```suggestion
:::tip
Astro now has support for [MDX](https://mdxjs.com/) through our [MDX integration](https://github.com/withastro/astro/tree/main/packages/integrations/mdx). MDX gives you the ability to include both Markdown and imported components in the same file. MDX can be good alternative for the `<Markdown />` component due to its large community and stable APIs.
:::
```

Wondering whether this might be a good "aside"?

I did also condense this a little for readablility, which admittedly loses a bit of the nuance but might be good enough for a note in the migration guide?